spring-cloud-gcp-data-firestore/src/main/java/org/springframework/cloud/gcp/data/firestore/FirestoreTemplate.java

```diff
@@ -54,6 +55,16 @@
  */
 public class FirestoreTemplate implements FirestoreReactiveOperations {
 
+	private static final String NAME_FIELD = "__name__";
+
+	private static final StructuredQuery.Projection ID_PROJECTION = StructuredQuery.Projection.newBuilder()
+			.addFields(StructuredQuery.FieldReference.newBuilder().setFieldPath(NAME_FIELD).build())
+			.build();
+
+	private static final DocumentMask NAME_ONLY_MASK = DocumentMask.newBuilder().addFieldPaths(NAME_FIELD).build();
+
+	private static final String NOT_FOUND_DOCUMENT = "NOT_FOUND: Document";
```

Comment:
Maybe rename to `NOT_FOUND_DOCUMENT_MESSAGE` to reflect what the constant is used for.